### DISCUSSION TITLE
Can we plot multiple auc for c3d,r3d101 and r3d152 in single ROC curve figure like shown in below figure?

### DISCUSSION THREAD

**Discussion by prasaddev9** [ekosman/AnomalyDetectionCVPR2018-Pytorch#169] [Ideas]
![1_aq0sX1-3hxRSl_GD7g4qag](https://user-images.githubusercontent.com/108169138/220649809-62608d1a-f2bc-4366-9b49-5e632fc1c699.png)

**Comment by ekosman** [comment]
Currently it is not possible and I don't plan to add this feature. However, I would welcome this as a contribution.
